src/EnergyPlus/EconomicTariff.cc

```diff
@@ -2711,7 +2711,7 @@ void ComputeTariff(EnergyPlusData &state)
 
     Real64 annualAggregate;
 
-    if (!(state.files.outputControl.tabular || state.files.outputControl.sqlite)) {
+    if (!state.files.outputControl.writeTabular(state)) {
```

Comment:
Dumb question here...why is the `state.dataOutRptTab->WriteTabularFiles` being set here in a function called `ComputeTariff` inside `EconomicTariff.cc` anyway??  That alone seems _super_ weird.

Reply:
It is a good question. And I don't remember, I suppose this might be the first place that a call to something dependent on actually writing tabular reports occurs, although that does not seem likely.